packages/hooks/src/useOnKeyDown/useOnKeyDown.mdx

```diff
@@ -0,0 +1,36 @@
+---
+name: useOnKeyDown
+menu: Hooks
+route: /hooks/use-on-key-down
+---
+
+import { Playground } from "docz";
+import { useState } from "react";
+import { useOnKeyDown } from "./useOnKeyDown";
+import { Text } from "@jobber/components/Text";
+
+# UseOnKeyDown
+
+`useOnKeyDown` is an extremly simple hook that can be for adding the keydown event handler when the component is mounted and remove them when it unmount.
+
+`useOnKeyDown` should **only** only by used when making keyboard shortcut in any component.
+
+```tsx
+import { useOnKeyDown } from "@jobber/hooks";
+```
+
+
+<Playground>
+  {()=>{
+    const letterText = "Delete me by pressing Delete key on your keyboard.";
```

Comment:
Macs label backspace and delete quite oddly which make this example a bit confusing for users on that platform. We can probably get a way with simplifying this just down to a single example. How would you feel about removing this delete one and switching your second one to `escape` `space-enter` and something like `ctrl-i` or something?